common/ferrostar/src/navigation_controller/mod.rs

```diff
@@ -155,7 +156,7 @@ impl NavigationController {
                         TripState::Navigating {
                             current_step_geometry_index: *current_step_geometry_index,
                             snapped_user_location: *snapped_user_location,
-                            remaining_steps,
+                            remaining_steps: remaining_steps.clone(),
```

Comment:
Why is this clone now needed?

Reply:
Eesh; you have the eyes of a hawk 😂 This was a remnant of an experiment in exposing the road name at the top level of `TripState`, which I abandoned after realize that it was literally a silly copy when the data was easily accessible. I undid that, but missed the extraneous clones you found.